src/lib/Preview.js

```diff
@@ -1051,6 +1061,10 @@ class Preview extends EventEmitter {
             case VIEWER_EVENT.mediaEndAutoplay:
                 this.navigateRight();
                 break;
+            case VIEWER_EVENT.error:
```

Comment:
Can we remove this case entirely?

Reply:
We cannot. This prevents a double error message from being emitted, which would double up on firing off preview_error events.